File: Templates/scripts/session_number.js
function session_number(){
	file = fopen('session_number.txt', 0);
	number = fread(file,flength(file));
	newNumber = number + 1;
	file = fopen('session_number.txt', 3);
	fwrite(file, newNumber);
	return newNumber;
}

module.exports = session_number;

File: Templates/scripts/session_number.py
import sys
import traceback
from pathlib import Path

template_folder = 'Templates'


def main():
    if sys.stdout.encoding != 'utf-8':
        sys.stdout.reconfigure(encoding='utf-8')
    path = Path(template_folder) / "session_number.txt"
    file = open(path, "w+")
    number = file.read()
    newNumber = f"{number + 1}"
    print(newNumber)
    file.write(newNumber)
    file.close()


if __name__ == '__main__':
    try:
        main()
    except Exception:
        print("Error when running user template.")
        print("```")
        traceback.print_exc(file=sys.stdout)
        print("```")
